packages/router-plugin/package.json

```diff
@@ -122,6 +122,7 @@
   },
   "peerDependencies": {
     "@rsbuild/core": ">=1.0.2",
+    "@tanstack/react-router": "workspace:^",
```

Comment:
why is this required? there is nothing used from that package in router-plugin?

Reply:
I believe it's because the router generator is declared as a dependency and it has that peer dependency declared, so this needs to forward that along.

Reply:
interesting that this is not handled automatically...

can you please add 

```json 
"peerDependenciesMeta": {
    "@tanstack/react-router": {
      "optional": true
    }
  }
  ```

Reply:
Sure!